### PR TITLE
Create new prowbazel image for bazel 0.22

### PR DESCRIPTION
**Pull request by sebastienvas**
Create new prowbazel image for bazel 0.22 and update proxy jobs on master to use it.

### CODE REVIEW

**Comment by sebastienvas** [comment]
cc @PiotrSikora

File: prow/cluster/jobs/istio/proxy/istio.proxy.master.yaml [hunk 1]
```diff
@@ -1,7 +1,7 @@
```
Comment (PiotrSikora) [inline prow/cluster/jobs/istio/proxy/istio.proxy.master.yaml:1]: The same should be done for release-1.1 branch.

Comment (utka) [reply prow/cluster/jobs/istio/proxy/istio.proxy.master.yaml:1]: https://github.com/istio/test-infra/pull/1170

**Pull request merged**